Dockerfile

```diff
@@ -11,13 +11,18 @@ RUN apk --no-cache add \
 	make
 
 ENV COREDNS_VERSION v1.6.9
+# TODO: change to "main" or tagged version
+ENV DNSSEEDER_VERSION addrv2
```

Comment:
We might want to use `main`, so we automatically get updates.

Otherwise it's a bit confusing if you push updates, but nothing happens in Docker.

Reply:
```suggestion
ENV DNSSEEDER_VERSION main
```